docs/cli.md

```diff
@@ -81,6 +81,7 @@ You can also find information on flags with `promscale_<version> -help`.
 
 | Flag | Type | Default | Description |
 |------|:-----:|:-------:|:-----------|
+| startup.dataset.config | string | | Dataset configuration in YAML format for Promscale. It is used for setting various dataset configuration like default metric chunk interval |
```

Comment:
Can we reference default values in the default column, so the user understands does the default values serve his/her requirements.

Referencing the `dataset.md` file that you created will be a good option to understand the default values. I would also suggest creating a table or a section with default values in `dataset.md` for quick visibility.